src/common/Microsoft.DotNet.UpgradeAssistant.Abstractions/MaybeNullWhenAttribute.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    [AttributeUsage(AttributeTargets.Parameter, Inherited = false)]
+    internal sealed class MaybeNullWhenAttribute : Attribute
```

Comment:
Does adding our own `[MaybeNullWhen]` like this work for satisfying NRT checks? I've never tried filling in missing types like this before.

Reply:
Yes it does (I have with some other projects).

Reply:
Very cool. Good to know about.